### PR TITLE
Add MSGraph provider for Sharepoint settings

### PR DESCRIPTION
**Pull request by brad-Simeon**

### CODE REVIEW

File: Source/Resources/Content/SharePoint/TenantProperties/Configuration.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
   "BlockDownloadOfAllFilesOnUnmanagedDevices": false,
   "BlockDownloadOfViewableFilesForGuests": false,
   "BlockDownloadOfViewableFilesOnUnmanagedDevices": false,
-  "BlockMacSync": false,
```
Comment (ashek-simeon) [inline Source/Resources/Content/SharePoint/TenantProperties/Configuration.json:18]: I was thinking we'd remove these in a separate PR after I merge changes into M365Management

**Review by ashek-simeon — DISMISSED** [review]
👍

File: Source/Resources/Content/MSGraph/Admin/Sharepoint/Settings/Configuration.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "isSharePointMobileNotificationEnabled": true,
+  "isSharePointNewsfeedEnabled": false,
+  "isSiteCreationEnabled": true,
+  "isSiteCreationUIEnabled": true,
+  "isSitePagesCreationEnabled": true,
+  "isSitesStorageLimitAutomatic": true,
+  "isSyncButtonHiddenOnPersonalSite": false,
+  "isUnmanagedSyncAppForTenantRestricted": true,
+  "personalSiteDefaultStorageLimitInMB": 1048576,
+  "sharingAllowedDomainList": [
```
Comment (ashek-simeon) [inline Source/Resources/Content/MSGraph/Admin/Sharepoint/Settings/Configuration.json:30]: What's going on with these 2?

File: Source/Resources/Content/MSGraph/Admin/Sharepoint/Settings/Configuration.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "domain1.com",
+    "domain2.com"
+  ],
```
Comment (ashek-simeon) [inline Source/Resources/Content/MSGraph/Admin/Sharepoint/Settings/Configuration.json:33]:
```suggestion
  "sharingAllowedDomainList": [],
```

**Review by ashek-simeon — APPROVED** [review]

**Review by mzorzella-simeon — APPROVED** [review]

**Pull request merged**